game_board.gd

```diff
@@ -134,3 +135,6 @@ func _on_victory():
 
 func _on_game_over():
 	$WinLoseLabel.text = "You lose :("
+
+func _on_ui_main_menu_selected():
+	main_menu_selected.emit()
```

Comment:
from what I can tell the only purpose of this signal is to send another signal. Why not listen to the original one directly?

Reply:
Since main menu button is a child of UI, we don't know of certain if it exists outside of UI. For example, Game Board does not statically know the children of UI, so I figured it would be less coupled to main menu button if it instead listens for what it knows exists, which is UI. Passing a signal up through a chain of children. This would allow the UI to be reworked and reorganized and everything work as long as UI handled when to send that signal out.

Reply:
game_board can and does access children of UI, if that's what you mean, like this line:
`for squad_button in $UI/RightSidebar/SquadDisplay.get_children():`

not worried about it though

Reply:
Good eye, that should probably be decoupled eventually